openshift-hack/cmd/k8s-tests-ext/disabled_tests.go

```diff
@@ -68,6 +67,11 @@ func filterOutDisabledSpecs(specs et.ExtensionTestSpecs) et.ExtensionTestSpecs {
 			// host. Enabling the test would result in the  bastion being created for every parallel test execution.
 			// Given that we have existing oc and WMCO tests that cover this functionality, we can safely disable it.
 			"[Feature:NodeLogQuery]",
+
+			// volumegroupsnapshot in csi-hostpath tests requires changes in the test yaml files,
+			// which are done by a script upstream. In OCP, we added a separate driver csi-hostpath-groupsnapshot,
+			// that will not be skipped by any rule here.
+			`[Driver: csi-hostpath].*[Feature:volumegroupsnapshot]`,
```

Comment:
The `*` wildcard here won't work. The OTE filtering mechanism doesn't use regex, it just looks for the value in the string.

Reply:
I think it does work, you can see `[sig-storage] In-tree Volumes [Driver: hostPath] [Testpattern: (delete policy)] volumegroupsnapshottable [Feature:volumegroupsnapshot] VolumeGroupSnapshottable should create snapshots for multiple volumes in a pod [Suite:openshift/conformance/parallel] [Suite:k8s]` skipped in a TP job [here](https://prow.ci.openshift.org/view/gs/test-platform-results/logs/openshift-kubernetes-2254-ci-4.19-e2e-aws-ovn-techpreview/1906723493894950912)

Reply:
/payload-job periodic-ci-openshift-release-master-ci-4.19-e2e-aws-ovn-techpreview

Reply:
So it seems that removal of the rule is all I need.

We want `[Driver: csi-hostpath].*volumegroupsnapshottable` to be skipped and it already has `[Disabled:SpecialConfig]'.
    
We want `[Driver: csi-hostpath-groupsnapshot].*volumegroupsnapshottable to run in TP jobs and it already has `[OCPFeatureGate:VolumeGroupSnapshot]...[Suite:openshift/conformance/parallel]`

I reworked this PR to remove the rule in k8s-tests-ext.

Reply:
I think the following, added to the `SpecialConfig` section of `disabled_tests.go`, will be sufficient:
`"[Driver: csi-hostpath] [Testpattern:  (delete policy)] volumegroupsnapshottable [Feature:volumegroupsnapshot]"`